triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/process/AllowFiltering.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger.simulation.process
+
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.actor.typed.scaladsl.Behaviors
+import com.daml.lf.engine.trigger.TriggerMsg
+
+object AllowFiltering {
```

Comment:
This wrapper process allows transaction messages to be filtered in a more granular manner